readthedocs/projects/forms.py

```diff
@@ -473,6 +473,12 @@ def clean_url(self):
             raise forms.ValidationError(
                 _('This field is required.')
             )
+        parsed_url = urlparse(url)
+
+        if parsed_url.scheme != 'https':
```

Comment:
I can also change this to look for other schemes like 'ftp' and 'ftps'.

Reply:
Let's keep the `http` protocol. `not in ('http', 'https')`